NMRangeSlider/NMRangeSlider.h

```diff
@@ -53,6 +53,9 @@
 // minimum value for right thumb
 @property(assign, nonatomic) float upperMinimumValue;
 
+// track height
+@property(assign, nonatomic) float thickness;
```

Comment:
I have it as a float right now, but yeah should be an int. I don't really see much of a use case for a float. I just changed my local copy to use an int and it worked just the same.